src/main/java/build/buildfarm/instance/memory/ByteStringStreamSource.java

```diff
@@ -45,15 +44,15 @@ public void write(byte[] b) {
 
       @Override
```

Comment:
now that it no longer extends anything, you need to remove the @Override from everywhere

Reply:
and I am unimpressed that that did not warn me! thanks!

Reply:
Ah, now I recall, note that this is in the anonymous override of InputStream and OutputStream (@Override does error in the line 75 case below

Reply:
Ah, right, sorry!